core/src/main/scripts/importer/validateData.py

```diff
@@ -4036,7 +4036,7 @@ def main_validate(args):
     text_handler.setFormatter(
         cbioportal_common.LogfileStyleFormatter(study_dir))
     collapsing_text_handler = cbioportal_common.CollapsingLogMessageHandler(
-        capacity=1e6,
+        capacity=5e5,
```

Comment:
Now that we actually run into this limitation, this is a good time to re-evaluate [the idea to get rid of buffering before collapse altogether](https://github.com/cBioPortal/cbioportal/blob/v1.14.0/core/src/main/scripts/importer/cbioportal_common.py#L425), monoidally. Hitting the buffer capacity currently means that the message will appear in the output multiple times, being collapsed in batches. But for the short term, this seems like a good fix for the Datahub CI issues as the number was always arbitrary anyway.